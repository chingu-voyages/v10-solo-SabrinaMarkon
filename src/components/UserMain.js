import React, { Component } from 'react';

export default class UserMain extends Component {


  render() {
    return (
      <div>
        <h1>UserMain</h1>
      </div>
    );
  }
}