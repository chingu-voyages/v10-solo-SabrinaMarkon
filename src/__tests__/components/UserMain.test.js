import React from 'react';
import { shallow } from 'enzyme';

// components
import UserMain from '../../components/UserMain';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<UserMain />);
});

describe('Testing UserMain component', () => {
  it('should render an h1 heading with the correct text', () => {
    expect(wrapper.find('h1').text().toEqual('UserMain'));
  });
  it('it should match the UserMain snapshot', () => {
    // If snapshot exists, compare. If not, create the snapshot.
    expect(wrapper).toMatchSnapshot();
  });
});