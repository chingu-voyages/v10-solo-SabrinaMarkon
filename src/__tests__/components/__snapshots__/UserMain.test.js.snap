// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UserMain component it should match the UserMain snapshot 1`] = `undefined`;
